src/js/components/Search/SearchResults.js

```diff
@@ -32,29 +32,30 @@ const SearchResults = () => {
 
   return (
     <div style={wrapperStyle}>
-      <Card 
+      <Card
         style={{ borderRadius: '10px', padding: '10px 33px', width: '1200px', minHeight: '28rem' }}
-        loading={isFetchingData}>
+        loading={isFetchingData}
+      >
         <Row gutter={16}>
           <Col span={4}>
             <Space direction="vertical" size="middle" style={{ display: 'flex' }}>
               <Statistic
                 title={t('Individuals')}
                 value={isValid ? (status === 'count' ? count : t(message)) : '----'}
-                valueStyle={{ color: '#1890ff' }}
+                valueStyle={{ color: '#75787a' }}
```

Comment:
This color value is used as a literal in 4 different places, we could put it in a constant and reference it instead.

Reply:
ah good point, 2nd this

Reply:
would be in the new pr oops, I have made the change in my current branch so whenever I merge it should be in it